src/main/scala/org/constellation/p2p/Cluster.scala

```diff
@@ -400,8 +409,12 @@ class Cluster[F[_]: Concurrent: Logger: Timer: ContextShift](ipManager: IPManage
         .map(v => keys.zip(v).toMap)
     } yield res
 
-  def join(hp: HostPort): F[Response[Unit]] =
-    attemptRegisterPeer(hp)
+  def join(hp: HostPort): F[Unit] = {
+    implicit val sDAO: DAO = dao
```

Comment:
sDAO what does mean this variable?

Reply:
shaded dao, basically I don't know to make an implicit for the whole class, just in the place where it's really needed